pkg/ebpf/tracer_windows.go

```diff
@@ -43,29 +37,52 @@ func init() {
 	}
 }
 
+// readBuffer is the buffer to pass into ReadFile system call to pull out packets
+type readBuffer struct {
+	ol   windows.Overlapped
+	data [128]byte
```

Comment:
Do these fields need to be layed out next to eachother  for overlapped IO to work? If so, can we document that so it doesn't get inadvertently refactored

Reply:
I think we make an assumption that this is the case, so I will add a comment.